index.html

```diff
@@ -5819,6 +5829,52 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________*CZ's card END ________  -->
 
+        <!-- ________ Aman Contributor card START ________  -->
```

Comment:
Thank you for the contribution, I won't be able to mark it with the tag as we are not in hacktoberfest but still contribution and collaboration is what matters.

Looks like you have added your card at the end of the file, you have to add your card just below the template as per instructions given in step 5, you can check it [here](https://github.com/Syknapse/Contribute-To-This-Project/).
Just do the changes and then we'll review the PR and merge it.